.github/workflows/release-drafter.yml

```diff
@@ -0,0 +1,32 @@
+name: Release drafter
+
+# Push events to every tag not containing "/"
+on:
+  push:
+    tags:
+      - "*"
+
+jobs:
+  draft-a-release:
+    name: Draft a release
+    runs-on: ubuntu-latest
```

Comment:
Thanks @Yury-Fridlyand Looks good!
One minor recommendation, there is a 2PR approval process we recently on added (that's why missing from jdbc repo I believe).
Can you add that as well?
Sample code: https://github.com/opensearch-project/spring-data-opensearch/blob/main/.github/workflows/release-drafter.yml#L16-L27

Sorry about that not sure how I missed it.

Reply:
Added in 1751721